app/views/works/index.html.erb

```diff
@@ -0,0 +1,76 @@
+<h1>List of Works</h1>
+
+<h2>Movies</h2>
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">Votes</th>
+      <th scope="col">Title</th>
+      <th scope="col">Created By</th>
+      <th scope="col">Published</th>
+      <th scope="col">Upvote</th>
+    </tr>
+  </thead>
+  <tbody>
+  <%@movies.each do |movie|%>
+    <tr>
+      <td><%=movie.votes.count%></td>
+      <td><%= link_to movie.title, work_path(movie)%></td>
+      <td><%=movie.creator%></td>
+      <td><%=movie.publication_year%></td>
+      <td><%=link_to "upvote", upvote_path(movie), :method => :post, :class => "btn btn-primary"%></td>
+    </tr>
+  <%end%>
+  </tbody>
+</table>
+
+
+<h2>Albums</h2>
+<table class="table">
+  <thead>
+    <tr>
+
+      <th scope="col">Votes</th>
+      <th scope="col">Title</th>
+      <th scope="col">Created By</th>
+      <th scope="col">Published</th>
+      <th scope="col">Upvote</th>
+    </tr>
+  </thead>
+  <tbody>
+  <%@albums.each do |album|%>
+    <tr>
+      <td><%=album.votes.count%></td>
+      <td><%= link_to album.title, work_path(album)%></td>
+      <td><%=album.creator%></td>
+      <td><%=album.publication_year%></td>
+      <td><%=link_to "upvote", upvote_path(album), :method => :post, :class => "btn btn-primary"%></td>
+    </tr>
```

Comment:
This code gets repeated. Is there a way you could use a view partial to Dry this up?